packages/figma-fetcher/src/downloadImage.js

```diff
@@ -37,7 +37,7 @@ export async function downloadImage(config, icon) {
           },
         },
       ],
-    }).data.replace(new RegExp(escapeRegExp(config.systemColor)), 'currentColor');
+    }).data.replaceAll(config.systemColor, 'currentColor');
```

Comment:
can we turn back new RegExp(escapeRegExp(config.systemColor)) because in other case user can pass color with [ or , and it will broke all script

Reply:
```ts
    const response = await fetch(url);
    const svgText = await response.text();
    const optimizedSvg = optimize(svgText, {
      plugins: [
        'preset-default',
        'removeDimensions',
        {
          name: 'addClassesToSVGElement',
          params: {
            classNames: [config.className],
          },
        },
      ],
    });

    const systemColorRegex = new RegExp(escapeRegExp(config.systemColor), 'g');
    const content = optimizedSvg.data.replace(systemColorRegex, 'currentColor');

```

mb something like it

Reply:
fixed